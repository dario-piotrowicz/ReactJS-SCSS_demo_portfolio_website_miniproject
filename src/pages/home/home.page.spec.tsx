import React from 'react';
import { shallow } from 'enzyme';
import HomePage from './home.page.component';

describe('home page', () => {
  it('should match its snapshot', () => {
    expect(shallow(<HomePage />)).toMatchSnapshot();
  });
});
