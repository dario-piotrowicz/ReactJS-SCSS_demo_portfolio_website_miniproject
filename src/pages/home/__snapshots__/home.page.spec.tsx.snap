// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`home page should match its snapshot 1`] = `
<Fragment>
  <SpecializeSection />
  <StatsSection />
  <CreationProcessSection />
</Fragment>
`;
